### PR TITLE
DOMA-2249 added request for consent on personal data proceeding on registration form

### PR DESCRIPTION
**Pull request by akarjakin**
*) Added request for user consent on proceeding personal data

image
image

*) Some minor code refactoring

### CODE REVIEW

File: apps/condo/domains/user/components/auth/InputPhoneForm.tsx [hunk 1]
```diff
@@ -94,57 +140,41 @@ export const InputPhoneForm: React.FC<IInputPhoneFormProps> = ({ onFinish })=> {
             labelAlign='left'
             requiredMark={false}
         >
-            <Row gutter={[0, 60]}>
+            <Row gutter={GUTTER_0_60}>
                 <Col span={24}>
-                    <Row gutter={[0, 40]}>
+                    <Row gutter={GUTTER_0_40}>
```
Comment (pahaz) [inline apps/condo/domains/user/components/auth/InputPhoneForm.tsx:145]: It's better to leave such consts here. It's not in increase the readability

Comment (akarjakin) [reply apps/condo/domains/user/components/auth/InputPhoneForm.tsx]: removed all refactoring

File: yarn.lock [hunk 2]
```diff
@@ -1997,10 +1997,6 @@
   version "0.0.0"
   uid ""
 
-"@condorb/domains@link:./apps/condorb/domains":
```
Comment (pahaz) [inline yarn.lock:2000]: why does the yarn lock change?

Comment (akarjakin) [reply yarn.lock]: removed

**Review by pahaz — APPROVED** [review]

**Review by sitozzz — APPROVED** [review]

**Pull request merged**